backend/ops_api/ops/utils/cans.py

```diff
@@ -87,26 +52,34 @@ def get_can_funding_summary(can: CAN, fiscal_year: Optional[int] = None) -> CanF
 
         total_funding = sum([c.budget for c in can.funding_budgets if c.fiscal_year == fiscal_year]) or 0
 
-        new_funding = get_new_funding_by_fiscal_year(can, fiscal_year)
+        new_funding = (
+            sum([c.budget for c in can.funding_budgets if c.fiscal_year == fiscal_year and can.active_period == 1]) or 0
+        )
 
         planned_funding = get_funding_by_budget_line_item_status(can, BudgetLineItemStatus.PLANNED, fiscal_year)
         obligated_funding = get_funding_by_budget_line_item_status(can, BudgetLineItemStatus.OBLIGATED, fiscal_year)
         in_execution_funding = get_funding_by_budget_line_item_status(
             can, BudgetLineItemStatus.IN_EXECUTION, fiscal_year
         )
         in_draft_funding = get_funding_by_budget_line_item_status(can, BudgetLineItemStatus.DRAFT, fiscal_year)
+
```

Comment:
Similar to the above, if `fiscal_year > can.funding_details.fiscal_year` and `fiscal_year == can.funding_budgets[x].fiscal_year` then `can.funding_budgets[x]` is considered "carry forward" - defined here as the budget with a fiscal year that is after the appropriation year `can.funding_details.fiscal_year`